os-keepalived.te

```diff
@@ -57,3 +58,7 @@ allow keepalived_t self:process setpgid;
 # Bugzilla #1434826
 allow keepalived_t ifconfig_exec_t:file entrypoint;
 sysnet_domtrans_ifconfig(keepalived_t)
+
+# Bugzilla 1170238
+allow keepalived_t ping_exec_t:file { execute getattr read open execute_no_trans };
```

Comment:
You can use macro: netutils_exec_ping(keepalived_t)

Reply:
Thank you, I fixed it.